src/ec/packages/ec-component-message/message.story.js

```diff
@@ -15,77 +18,55 @@ import dataWarning from './demo/data--warning';
 import message from './ecl-message.html.twig';
 import notes from './README.md';
 
-const formatIcon = data => {
-  data.icon.path = defaultSprite;
-  data.close.icon.path = defaultSprite;
+const prepareMessage = data => {
+  data.variant = select(
+    'variant',
+    [data.variant],
+    data.variant,
+    buttonLabels.required
+  );
+
+  const name = select(
+    'icon.name',
+    [data.icon.name],
+    data.icon.name,
+    buttonLabels.required
+  );
+  getIconKnobs(data, name, 'notifications', 'l', 'primary', 'none');
+
+  data.title = text('title', data.title, buttonLabels.required);
+
+  data.description = text(
```

Comment:
Why don't we move these knobs on top? title and description i mean?

Reply:
@planctus I was following same order as https://github.com/ec-europa/ecl-twig/blob/4b67ca05606c3444040e1be7bc7290af40ee64c9/src/ec/packages/ec-component-message/ecl-message.html.twig#L5

But yes maybe, it better to have them on top?

Reply:
ah...i see the criteria, but yes i think we want them on top regardless of how we wrote this "example".. :)

Reply:
ok make sense, PR updated ;)